wolfcrypt/src/port/devcrypto/wc_devcrypto.c

```diff
@@ -122,7 +122,7 @@ int wc_DevCryptoCreate(WC_CRYPTODEV* ctx, int type, byte* key, word32 keySz)
         case CRYPTO_SHA2_512_HMAC:
             ctx->sess.cipher = 0;
             ctx->sess.mac    = type;
-            ctx->sess.mackey    = (uint8_t*)key;
+            ctx->sess.mackey    = (u_int8_t*)key;
```

Comment:
for older compiler versions this was switched to u_int8_t

Reply:
BTW this [change broke compilation](https://github.com/wolfSSL/wolfssl/issues/5693#issuecomment-1277340109) on systems using `musl` libc as it needs explicit `#include <sys/types.h>` to have this type available.

Reply:
Thanks @ynezz for the report and analysis, am getting back to this now to work on a resolution. Apologies for the delay and hassle that the change caused.